ts/parts/Pointer.ts

```diff
@@ -1283,15 +1287,21 @@ class Pointer {
         // Normalize before the 'if' for the legacy IE (#7850)
         e = this.normalize(e);
 
-        if ((e as any).button !== 2) {
+        this.onContainerMouseMove(e);
 
-            this.zoomOption(e);
+        if (
+            typeof e.button === 'undefined' ||
+            e.button === 0 // #11635, limiting to primary button
```

Comment:
This logic now changed from `e.button !== 2` to `e.button === 0`. If I remember correctly, the primary button in IE is or was 1, hence the !== 2 part. Has this been tested with IE? If so, we should have a comment.

Reply:
Just to make sure, I understand the issue, the original condition was already flawed, correct? So we need additionally some IE specific code here?